README.md

```diff
@@ -1,15 +1,105 @@
+* **Describe the behavior you observed after following the steps** and explain exactly what the problem is with that behavior.
+* **Explain which behavior you expected instead** and why.
+* **Provide screenshots and/or screen recordings** that might help explain the issue you are facing. To screen record a phone connected to Android Studio or an emulator follow the steps [here](https://developer.android.com/studio/debug/am-video). To screen record on iOS follow the steps described [here](https://support.apple.com/en-us/HT207935).
+* **Include relevant logs in the bug report** by putting it in a [code block](https://help.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github#multiple-lines), a [file attachment](https://help.github.com/en/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests) or in a [gist](https://help.github.com/en/github/writing-on-github/creating-gists) and provide a link to that gist.
+* **Tell how recently you started having the issue.** When was the first time you experienced the issue and was it after updating the SDK version? Or has it always been a problem?
+* If the problem started happening recently, **can you reproduce it in an older version of the SDK?** What's the most recent version in which the problem doesn't happen?
+* **Can you reliably reproduce the issue?** If not, explain how often it occurs and under what conditions it normally happens. For example in what environment you are running.
+
+Include details about the device/emulator/simulator you are experiencing the issue on:
+
+* **Which version of the SDK are you using?**
+* **Which OS is this a problem in, iOS, Android or both?** What version(s)? Also add the appropriate label to the issue.
+* **Did you experience the issue in simulator/emulator or on real device(s)?**
+
+#### Contributing with Code
+Before contributing please read through the [Klarna In-App SDK documentation](https://developers.klarna.com/documentation/in-app/).
+
+##### Branching
+Prefix the branch you are going to work on depending on what you are working on (bug fix or feature). Use the following prefixes when creating a new branch:
+
+* **feature/** if the branch contains a new feature, example: `feature/my-shiny-feature`.
+* **bugfix/**  if the branch contains a bug fix, example: `bugfix/my-bug-fix`.
+
+##### Pull Requests
+When creating a PR include as much information as possible about the type of enhancement, whether if it's a bugfix, new functionality or any other change. There's [a template](https://github.com/klarna/klarna-mobile-sdk/blob/a55293185f73609df831a26ddff5a66033f21e55/.github/ISSUE_TEMPLATE/pull-request.md) for you to fill out which will make the review process for the maintainers faster. When creating a PR do it against the `master` branch. The PR should include:
```

Comment:
Same here, need a template for PR's in this repo

Reply:
@adrianpalmquist done.